apps/etterlatte-saksbehandling-ui/client/src/components/generellbehandling/Attesteringmodal.tsx

```diff
@@ -27,7 +27,9 @@ export const Attesteringmodal = (props: {
       <Button style={{ marginTop: '1rem' }} onClick={() => attesterWrapper()} loading={isPending(attesterStatus)}>
         Godkjenn kravpakken
       </Button>
-      {isSuccess(attesterStatus) && <Alert variant="success">Behandlingen ble attestert</Alert>}
+      {isSuccess(attesterStatus) && (
+        <Alert variant="success">Behandlingen ble attestert, vi sender deg straks til saksoversikten</Alert>
```

Comment:
"vi sender deg" 

Ikke så fryktelig nøye, men for meg føles det unaturlig å skrive "vi" i systemsammenheng. 
Gir mer mening å skrive generelt "du blir straks sendt".